kind-config.yaml

```diff
@@ -1,9 +1,9 @@
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  image: sonobuoy/kind-node:v1.19.0
+  image: &kind_node_image kindest/node:v1.19.3
```

Comment:
Why this change to `&kind_node_image kindest/node...`

Reply:
I thought it'd be more helpful to make sure that the `control-plane` and `worker` nodes have consistent versions, instead of manually ensuring they are the same.